orchagent/bufferorch.cpp

```diff
@@ -21,14 +19,39 @@ type_map BufferOrch::m_buffer_type_maps = {
     {APP_BUFFER_QUEUE_TABLE_NAME, new object_map()},
     {APP_BUFFER_PG_TABLE_NAME, new object_map()},
     {APP_BUFFER_PORT_INGRESS_PROFILE_LIST_NAME, new object_map()},
-    {APP_BUFFER_PORT_EGRESS_PROFILE_LIST_NAME, new object_map()}
+    {APP_BUFFER_PORT_EGRESS_PROFILE_LIST_NAME, new object_map()},
+    {APP_BUFFER_PORT_CONFIG_TO_PG_PROFILE_TABLE, new object_map()},
+    {APP_BUFFER_PORT_CABLE_LENGTH_TABLE, new object_map()}
 };
 
-BufferOrch::BufferOrch(DBConnector *db, vector<string> &tableNames) : Orch(db, tableNames)
+BufferOrch::BufferOrch(DBConnector *db, vector<string> &tableNames, PortsOrch *portsOrch) :
+    Orch(db, tableNames),
+    m_portsOrch(portsOrch),
+    m_db(APPL_DB, DBConnector::DEFAULT_UNIXSOCKET, 0)
 {
     SWSS_LOG_ENTER();
     initTableHandlers();
-};
+
+    m_portsOrch->attach(this);
+}
+
+void BufferOrch::update(SubjectType type, void *cntx)
+{
+    SWSS_LOG_ENTER();
+
+    assert(cntx);
```

Comment:
Is this assert part of test code? Can we remove it or replace with handling code?